site/content/en/reference/live/_index.md

```diff
@@ -20,4 +20,129 @@ description: >
 
 Live contains the next-generation versions of apply related commands for
 deploying local configuration packages to a cluster.
+
+### How to Upgrade to the Next Generation Inventory Object
+
+#### What is an Inventory Object
+
+An inventory object is the automatically generated object which keeps track
+of the set of objects applied together. The current inventory object type
+is a ConfigMap, and it is usually defined in a package file called
+**inventory-template.yaml**. This file is created from an invocation of
+`kpt live init`. A typical use of the inventory object is to prune (delete)
+objects omitted locally.
+
+#### What is a Next Generation Inventory Object
+
+The next generation inventory object is a **ResourceGroup** custom resource
+replacing the current ConfigMap. Because the new inventory object is a
+custom resource, you must have permissions to add a custom resource
+definition (CRD) to the cluster. The individual command to add the
+**ResourceGroup** CRD is:
+
+```
+export RESOURCE_GROUP_INVENTORY=1
+kpt live install-resource-group
+```
+
+#### Upgrade Scenario 1: New (Uninitialized) Packages
+
+Packages which are newly downloaded and uninitialized should follow the
+following steps:
+
+1. `export RESOURCE_GROUP_INVENTORY=1`
+2. `kpt live init <PACKAGE DIR>`
+3. `kpt live install-resource-group`
```

Comment:
Step 3 is automatically done by `kpt live apply` right? I think we talked about having this explicit command as a fall out of `kpt live apply` failing to install the CRD for some reason.

Reply:
Fixed.